tools/git-sync-deps

```diff
@@ -37,14 +41,15 @@ Git Config:
       git config --unset sync-deps.disable
 """
 
-
 import os
 import re
 import subprocess
 import sys
 import threading
 from builtins import bytes
 
+with_clspv = False
+with_dxc = False
 
```

Comment:
Did you skip the swiftshader bits here and below on purpose?

Reply:
Nope adding now.